test_unit.py

```diff
@@ -1156,6 +1159,24 @@ def test_HTLC(self):
         self.assertEqual(p, base64.b64decode(golden))
         self.assertEqual(s.get_address(), golden_addr)
 
+    def test_periodic_fee(self):
+        addr = "726KBOYUJJNE5J5UHCSGQGWIBZWKCBN4WYD7YVSTEXEVNFPWUIJ7TAEOPM"
+        s = template.PeriodicPayment(addr, 10000, 999, 11, 10, 123456)
+        s.lease_value = "f4OxZX/x/FO5LcGBSKHWXfwtSx+j1ncoSt3SABJtkGk="
+
+        golden_addr = ("IJPPJDULMZNQXBIGCLCWVP5P4VD6U" +
+                       "564BHC7C5CMZB2FVXKGZSIVPJFPT4")
+
+        golden = ("ASAHAQoLAOcHkE7AxAcmAiB/g7Flf/H8U7ktwYFIodZd/C1LH6PWdyhK" +
+                  "3dIAEm2QaSD+vKC7FEpaTqe0OKRoGsgObKEFvLYH/FZTJclWlfaiEzEQ" +
+                  "IhIxASMOEDECJBglEhAxBCEEMQIIEhAxBigSEDEJMgMSMQcpEhAxCCEF" +
+                  "EhAxCSkSMQcyAxIQMQIhBg0QMQglEhAREA==")
+        p = s.get_program()
+        self.assertEqual(p, base64.b64decode(golden))
+        self.assertEqual(s.get_address(), golden_addr)
+        sk, pk = account.generate_account()
```

Comment:
what is the generated account used for here?

Reply:
Since we don't have a golden for the created transactions, I'm just using this to make sure the get_transactions() function doesn't error.

Reply:
What I mean is, `s.get_withdrawal_transaction(1234, 2234, "f4OxZX/x/FO5LcGBSKHWXfwtSx+j1ncoSt3SABJtkGk=", 10)` doesn't seem to use `sk` or `pk`

Reply:
You're right! My mistake.